README.md

```diff
@@ -119,6 +119,54 @@ Student has an attribute for the user, user has an attribute for the student, ..
 These circular references will prevent the log entry from being written.
 Including request is always safe, because the logging formatter contains dedicated code for request logging.
 
+## Javascript & CSS
+
+This project uses webpack to create javascript bundles. Custom Javascript is only added to pages when it is needed to enhance default Django functionality or to create user experience improvements.
+
+Notable examples are
+- Map Views
+- User Profile to hide unneeded blocks
+
+Javascript sources are located in frontend/src folder
+
+For local development node needs to be installed. For pure python development alternatively Docker can be used to build the javascript bundles. At the moment only production builds are supported via docker.
+
+### Build via docker
+
+Run the following commands in the main directory to setup the docker image
+```
+cd frontend
+docker build -t frontend:latest .
+```
+
+Run `docker run -ti -v "$(pwd)/dist:/app/dist" --rm frontend:latest` to build the bundles. They will then be available under the dist directory. Docker-Compose files for the main app use volumes instead to share the files between containers.
+
+### Build (local install) dev & prod
+
+To build locally node needs to be installed. We suggest using [Node Version Manager](https://github.com/nvm-sh/nvm) to install node. Dependencies can be installed using `cd frontend && npm install`
```

Comment:
I like to supply a working dev environment but most people won't use it, so it's debatable

Reply:
I would appreciate it as well :) Even when you're not using it most of the time, it seems invaluable to me for new forks.

Reply:
Hmmm my intention was to describe what you would need to do, to setup a local environment while also documenting the dockerized way. Currently both are possible:
- install node and build a bundle in your local dev environment, which is usually preferred. If you have node already it works quite well and handles different dependencies in multiple projects well
- If you don't want to have a local node setup you can use the provided Dockerfile. If you run it in dev mode it bind mounts the JS source and will rebuild automatically on every change. There are some features (hotreloading the code) which I don't plan to support in Docker as I don't want to spend that additional time.

I checked that both ways work, is the documentation unclear?

Reply:
Hm, I have to say as someone who has barely worked with node before, I would be really unsure what to install here in which scenario. I don't really have a better suggestion, but this does not work for me. One solution could be to move the node stuff closer to the "normal" setup.

Possible open questions after reading this are:
- Do I need to install node when I want to develop a new version, but will only change django/python stuff in order to build the packs?
- What is the difference between js dev and production builds? (_At the moment only production builds are supported via docker._)
- Do I need to call special commands in order to compile JS?

Reply:
>  move the node stuff closer to the "normal" setup.
Meaning?